src/viking.js

```diff
@@ -1,11 +1,97 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health,strength) {
+        this.health = health;
+        this.strength = strength;
+    };
+    attack() {
+        return this.strength;
+    };
+    receiveDamage(damage) {
+        this.health-=damage;
+    }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength) {
+        super(health,strength);
+        this.name=name;
+    };
+    attack() {
+        return this.strength;
```

Comment:
Since the class Viking extends the Soldier class, and the attack() method in the Viking class is exactly the same as the one in the Soldier class, you don't need to declare it again inside the Viking class because it will inherit this method from the Soldier class